crates/biome_css_formatter/tests/specs/css/issue_3229.css.snap

```diff
@@ -1,7 +1,6 @@
 ---
 source: crates/biome_formatter_test/src/snapshot_builder.rs
 info: css/issue_3229.css
-snapshot_kind: text
```

Comment:
It seems your local version of `cargo-insta` is outdated. You can update it using `just update-tools`, or just `cargo install cargo-insta`

Reply:
Thanks for the reminder, I have updated it